test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -86,7 +86,7 @@ it("GIVEN a sequencer, WHEN <correctly> canceling an update THEN a % of the slas
   );
   await waitSudoOperationSuccess(cancelResolutionEvents, "SudoAsDone");
   await waitBlockNumber(
-    (disputeEndBlockNumber2 + 1).toString(),
+    disputeEndBlockNumber2.toString(),
     disputePeriodLength * 2,
   );
```

Comment:
this is not deterministic anymore since one change Mat did, we need to use the waiter for the update executed. After that change, an update can be executed in the same block, the next, or whenever it fits some room.

Reply:
I understand that we can delete this one because we check txIndex later in the test. Fixed